lib/info.js

```diff
@@ -46,7 +47,20 @@ module.exports = function getInfo(link, options, callback) {
 
         // Get description from #eow-description
         var description = util.getVideoDescription(body);
-
+        var related = '';
+        var ids = '';
+        if( body.search(/RELATED_PLAYER_ARGS/) ) {
+          related = body.match(/RELATED_PLAYER_ARGS['\s\t\n]*:[\s\t\n]*{[\s\t\n"]*rvs":"([^}]+)/)[1];
```

Comment:
Tests fail from this line, make sure to check if there is a match.

I'm also not a big fan of this regexp, for one, singlequotes always surround `RELATED_PLAYER_ARGS` right? And `\s` also matches `\t` and `\n`. `rvs` is also always surrounde by doublequotes, it looks like.